src/__test__/util.ts

```diff
@@ -0,0 +1,10 @@
+import fetch from 'node-fetch'
+
+export async function getDataFromAPI (path: string) {
+  const response = await fetch(`https://explorer.cardano.org/api/${path}`)
```

Comment:
Shouldn't it be `api-new` in explorer url instead of `api` ?

Reply:
`api-new` uses the same data source: `cardano-db-sync`. Theses tests are checking for parity against the `cardano-sl` implementation